src/main/cpp/DragonVision/DragonLimelight.cpp

```diff
@@ -221,17 +221,38 @@ std::optional<units::angle::degree_t> DragonLimelight::GetTargetYaw()
 std::optional<units::angle::degree_t> DragonLimelight::GetTargetYawRobotFrame()
 {
     // Get the horizontal angle to the target and convert to radians
-    std::optional<units::angle::radian_t> limelightFrameHorizAngleRad = GetTargetYaw();
-    std::optional<units::length::inch_t> targetXdistance = EstimateTargetXDistance();
-    if (limelightFrameHorizAngleRad && targetXdistance)
-    {
-        units::length::inch_t targetHorizOffset = targetXdistance.value() * tan(limelightFrameHorizAngleRad.value().to<double>());
+    /* std::optional<units::angle::radian_t> limelightFrameHorizAngleRad = GetTargetYaw();
```

Comment:
Are we planning on re-activating this block of code?  If not, just delete it  

Several other blocks like this in this file

Reply:
Updated